### PR TITLE
SEO-177700-Dropped-keywords

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @gowthamanthavasiyappan ,
Please review the changes.
Thanks.
Regards,
Edith.

### CODE REVIEW

File: ej2-react/calendar/getting-started.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Calendar component | Syncfusion
-description:  Checkout and learn about Getting started with React Calendar component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with React Calendar Component | Syncfusion
+description:  Checkout and learn about getting started with Syncfusion Essential React Calendar component, it's elements and more details.
```
Comment (gowthamanthavasiyappan) [inline ej2-react/calendar/getting-started.md:4]: Revert this and remove unwanted space only.

File: ej2-react/image-editor/getting-started.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Image editor component | Syncfusion
-description:  Checkout and learn about Getting started with React Image editor component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with React Image Editor Component | Syncfusion
+description:  Checkout and learn about getting started with Syncfusion Essential React Image Editor component, it's elements and more details.
```
Comment (gowthamanthavasiyappan) [inline ej2-react/image-editor/getting-started.md:4]: Revert this, this is ej2 UG page, so the old description is fine.

File: ej2-react/uploader/getting-started.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Uploader component | Syncfusion
-description:  Checkout and learn about Getting started with React Uploader component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with React Uploader Component | Syncfusion
+description:  Checkout and learn about getting started with Syncfusion Essential React Uploader component, it's elements and more details.
```
Comment (gowthamanthavasiyappan) [inline ej2-react/uploader/getting-started.md:4]: Revert this, check space issue only.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: ej2-react/uploader/getting-started.md (hunk 3 above)
Comment (Edithabich) [reply ej2-react/uploader/getting-started.md]: resolved

**Pull request merged**